tests/test_reserved_terms.py

```diff
@@ -0,0 +1,17 @@
+from nomenclature import DataStructureDefinition
+
+
+def test_variable_ops_as_square_brackets():
+    # Check that variables use square brackets for operations
+    # https://github.com/IAMconsortium/common-definitions/issues/55
+
+    dsd = DataStructureDefinition("definitions/", dimensions=["variable"])
+
+    error = []
+    for variable, attrs in dsd.variable.items():
```

Comment:
I think you don't need `attrs` here:
```suggestion
    for variable in dsd.variable.keys():
```